.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,28 @@
+name: Deploy HeyEmoji to AWS using the BitOps Github Action
+
+on:
+  push:
+    branches:
+      - "main"
+  workflow_dispatch:
+    inputs:
+      aws-default-region:
+        type: string
+        description: AWS Region targeted for deployment
+        default: us-east-1
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - id: deploy
+      uses: bitovi/github-actions-node-app-to-aws-vm@main
```

Comment:
@PhillypHenning 
update to use new github action name like: https://github.com/bitovi/heyemoji/pull/1/files#diff-4f9e38227ed64fefb17f4668a7ac4ab55b6149994d5ac2fd96182d5958479b54R22

Reply:
Will do this morning